src/inputs/Button/Button.tsx

```diff
@@ -2,17 +2,22 @@ import { PropsWithChildren } from "preact/compat";
 import { h } from "preact";
 
 import * as styles from "./Button.module.css";
+import { Plus, Spinner } from "../../assets"
+
 
 interface ButtonProps {
   onClick(): void;
-  disabled?: boolean;
   variant?: "primary" | "secondary";
+  disabled?: boolean;
   isLoading?: boolean;
+  icon?: boolean;
```

Comment:
**issue:** What if you want to pass any other Icon into your button? In that case it is impossible - you should be able to pass any SVG icon as property